library/src/scala/quoted/Quotes.scala

```diff
@@ -4373,6 +4373,18 @@ trait Quotes { self: runtime.QuoteUnpickler & runtime.QuoteMatching =>
       /** Is this symbol `abstract` */
       def Abstract: Flags
 
+      /** Is this symbol is labeled with of abstract & override
+       *
+       *  The override modifier has an additional significance when combined with the abstract modifier.
+       *  That modifier combination is only allowed for value members of traits.
+       *
+       *  We call a member MM of a template incomplete if it is either abstract (i.e. defined by a declaration), or it is labeled abstract and override and every member overridden by MM is again incomplete.
+       *
+       *  Note that the abstract override modifier combination does not influence the concept whether a member is concrete or abstract.
+       *  A member is abstract if only a declaration is given for it; it is concrete if a full definition is given.
+       */
```

Comment:
I've rewritten the documentation to be more user-friendly
```suggestion
      /** Is this an abstract override method?
       *
       *  This corresponds to a definition declared as "abstract override def" in the source.
       * See https://stackoverflow.com/questions/23645172/why-is-abstract-override-required-not-override-alone-in-subtrait for examples.
       */
```